cli/src/main.rs

```diff
@@ -60,76 +60,90 @@ fn run<I: IntoIterator<Item = T>, T: Into<OsString> + Clone>(args: I) -> Result<
         (@arg quiet: -q --quiet +global "Do not display output")
         (@setting SubcommandRequiredElseHelp));
 
-    #[cfg(feature = "workload")]
+    #[cfg(any(feature = "workload", feature = "workload-smallbank"))]
```

Comment:
Should this just be on workload? (given that "workload-smallbank" depends on "workload"?

Reply:
Fixed the feature guards